tests/mock-snapd.c

```diff
@@ -1687,9 +1688,15 @@ static void mock_change_free(MockChange *change) {
   g_free(change->summary);
   g_free(change->spawn_time);
   g_free(change->ready_time);
-  g_list_free_full(change->tasks, (GDestroyNotify)mock_task_free);
-  if (change->data != NULL)
-    json_node_unref(change->data);
+#ifdef GLIB_VERSION_2_64
```

Comment:
If there are no SRU-concerns I think is totally fine to bump the glib dependency: 2.64 it's 4 years old already so I don't think we need to care about it too much.

Reply:
This was here so we could support 18.04 LTS, if that's no longer a concern then bumping it to 2.64 means the minimum supported version is 20.04 LTS (at least without patching to support 18.04).

Reply:
You are right. Removed the #ifdefs and left the code as before.